src/lib.rs

```diff
@@ -56,6 +56,7 @@ pub fn plugin_registrar(reg: &mut Registry) {
     reg.register_lint_pass(box misc::ModuloOne as LintPassObject);
     reg.register_lint_pass(box unicode::Unicode as LintPassObject);
     reg.register_lint_pass(box strings::StringAdd as LintPassObject);
+    reg.register_lint_pass(box strings::StringAddAssign as LintPassObject);
```

Comment:
(If something is done about that, there are two more lints that are `Allow` by default, `PTR_ARG` and `RESULT_UNWRAP_USED`.)